app.js

```diff
@@ -14,12 +14,12 @@ app.use(express.json());
 
 app.use("/api/contacts", contactsRouter);
 
-app.use((req, res) => {
-  res.status(404).json({ message: "Not found" });
+app.use((request, response) => {
```

Comment:
```suggestion
app.use((req, res) => {
```
Tak się przyjeło że się pisze.

Reply:
Oki, tak jak rozmawialiśmy na slacku, w kolejnych projektach będę się trzymał skróconej wersji, nie ma co iść pod prąd, zrobiłem to tylko dla czytelności dla samego siebie, pisanego przez siebie kodu :-)